modules/color-picker/react/lib/parts/ColorSwatch.tsx

```diff
@@ -11,6 +11,19 @@ export interface ColorSwatchProps extends React.HTMLAttributes<HTMLDivElement> {
   showCheck?: boolean;
 }
 
+// cache a div instance to decrease garbage collection
+const div = document.createElement('div');
+/**
+ * Use the browser's built-in mechanism for comparing colors. Works all the way back to IE11. The
+ * browser will take any valid color and convert to something like `rgb(0, 0, 0)` which allows us to
+ * then compare the strings even if the colors are in different formats
+ */
+function compareColors(color1: string, color2: string): boolean {
+  div.style.color = color1;
+  div.style.backgroundColor = color2;
+  return div.style.color === div.style.backgroundColor;
+}
```

Comment:
This is a clever hack to get the browser to do the color conversion work for us without requiring chromajs. If all we want to do is compare the same color, we can use this method.